### ISSUE TITLE
Error when executing Start.bat: "The member 'Pool' is already defined."

### ISSUE THREAD

**Issue by caniko2** [MultiPoolMiner/MultiPoolMiner#2372]
I get this error when executing the default start.bat file:
![image](https://user-images.githubusercontent.com/29519599/62936768-f3a17900-bdca-11e9-961d-f091c8c8a2f7.png)

```
& : At C:\Portable Apps\Mining\mpm\Include.psm1:1337 char:5
+     $Pool
+     ~~~~~
The member 'Pool' is already defined.
At line:1 char:3
+ & .\multipoolminer.ps1 -Wallet 1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb -Us ...
+   ~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (C:\Portable Apps\Mining\mpm\Include.psm1:String) [], RuntimeException
+ FullyQualifiedErrorId : MemberAlreadyDefined
```

**Comment by UselessGuru** [comment]
What version?
Did you start with a new directory?
Include.psm1 of version 3.5.0 does not contain a line 1337 with '$Pools'

**Comment by caniko2** [comment]
I pulled from the repository, and yes it is a new dir

**Comment by UselessGuru** [comment]
> I pulled from the repository,

From what?
The master code is not working for releases. They are all in the 'dev' branch.
https://github.com/MultiPoolMiner/MultiPoolMiner/tree/dev

**Comment by caniko2** [comment]
Oh, I didn't know that. I have been using the master branch since day one. What branch has the most stable release?

**Comment by UselessGuru** [comment]
> What branch has the most stable release?

I'd say 'dev'.

Aaronsace is working on something new in the 'master' branch (I know as much as you :-).
I agree with you that one would expect the master to be one to use, but this branch is currently broken. An I can only update the code in the 'dev' branch.

See also https://github.com/MultiPoolMiner/MultiPoolMiner/issues/2356#issuecomment-513498349

**Comment by caniko2** [comment]
I get this error in the dev branch before each refresh:
```
Conversion from JSON failed with error: Invalid property identifier character: ਍. Path '', line 2, position 0.
+ CategoryInfo          : NotSpecified: (:) [ConvertFrom-Json], ArgumentException
+ FullyQualifiedErrorId : System.ArgumentException,Microsoft.PowerShell.Commands.ConvertFromJsonCommand
+ PSComputerName        : localhost
```
Imagine 100 of copies of this message over and over again before each refresh.

**Comment by UselessGuru** [comment]
@caniko2 

Please post your config file. I suspect an issue with the parameter '-AllowedBadShareRatio'.
Please try to set this to a reasonable value, e.g. 0.2 (see the README for more information on this parameter)

**Comment by UselessGuru** [comment]
@caniko2 

The reported issue is fixed in version 3.5.1:
https://github.com/MultiPoolMiner/MultiPoolMiner/releases/tag/3.5.1
